lib/karafka/base_consumer.rb

```diff
@@ -32,6 +41,13 @@ def on_before_enqueue; end
     def on_before_consume
       messages.metadata.processed_at = Time.now
       messages.metadata.freeze
+    rescue StandardError => e
```

Comment:
Why don't we have `handle_before_consume` here?

Reply:
because we never needed it in any scenario. I will add it if there is a scenario that would need it.